tests/test_solvermode/test_boundaries.py

```diff
@@ -76,3 +80,52 @@ def test_boundaries_elbow(load_mixing_elbow_mesh):
         ].turb_viscosity_ratio()
         == 4
     )
+
+
+@pytest.mark.integration
+@pytest.mark.setup
+def test_boundaries_peridoicrot(load_peridoicrot_withpipes_cas):
+    session = load_peridoicrot_withpipes_cas
+    pysession = session.solver.root
+    boundary_exp = json.load(
+        open("tests\\test_solvermode\\boundaries_peridoicrot_expDict")
+    )
+    boundary_test = {}
+    boundary_tested = {}
+    for name, boundary in pysession.setup.boundary_conditions.items():
+        boundary_test[name] = boundary()
+    boundary_tested["val_1"] = boundary_test
+    TestCase().assertDictEqual(boundary_tested["val_1"], boundary_exp["val_1"])
+
+    boundary_test = {}
+    for boundary_type in pysession.setup.boundary_conditions.get_active_child_names():
```

Comment:
I wonder if it's worth testing the `get_active_child_names()` list?

Reply:
Yeah, I think it would be worth it.  It would be a test of the dependency logic right.